app/classes/api2/core/base.rb

```diff
@@ -213,41 +213,37 @@
 #  +localization_files_test.rb+ will automatically search through all the
 #  parsers looking for missing error message.
 #
-class API2
-  API_VERSION = 2.0
-
-  def self.version
-    API_VERSION
+module API2::Base
+  # When we `include` a module, `self` refers to the module not the class,
+  # so only instance methods are added. Add class methods like this:
```

Comment:
This comment is confusing, and I'm afraid it maybe misleading.  Inside a method self still refers to the instance, neither the class nor the module, right?